posix-spawn-0.3.8 fails to build with Alpine Linux and musl libc

```
Building native extensions.  This could take a while...
/usr/bin/ruby extconf.rb
creating Makefile
make "DESTDIR="
compiling posix-spawn.c
In file included from /usr/include/ruby-2.0.0/ruby/ruby.h:24:0,
                 from /usr/include/ruby-2.0.0/ruby.h:33,
                 from posix-spawn.c:14:
/usr/include/ruby-2.0.0/x86_64-linux-musl/ruby/config.h:17:0: warning: "_GNU_SOURCE" redefined [enabled by default]
 #define _GNU_SOURCE 1
 ^
posix-spawn.c:3:0: note: this is the location of the previous definition
 #define _GNU_SOURCE
 ^
posix-spawn.c: In function 'rb_posixspawn_pspawn':
posix-spawn.c:403:11: error: 'POSIX_SPAWN_USEVFORK' undeclared (first use in this function)
  flags |= POSIX_SPAWN_USEVFORK;
           ^
posix-spawn.c:403:11: note: each undeclared identifier is reported only once for each function it appears in
Makefile:223: recipe for target 'posix-spawn.o' failed
make: *** [posix-spawn.o] Error 1
ERROR:  Error installing /home/ncopa/aports/wip/ruby-posix-spawn/src/posix-spawn-0.3.8.gem:
    ERROR: Failed to build gem native extension.

    Building has failed. See above output for more information on the failure.
```

btw, why are the no git tag for 0.3.8 in https://github.com/rtomayko/posix-spawn/releases ?

--- a/ext/posix-spawn.c
+++ b/ext/posix-spawn.c
@@ -396,9 +396,9 @@ rb_posixspawn_pspawn(VALUE self, VALUE env, VALUE argv, VALUE options)
 	sigemptyset(&mask);
 	posix_spawnattr_setsigmask(&attr, &mask);
 
-#if defined(POSIX_SPAWN_USEVFORK) || defined(__linux__)
-	/* Force USEVFORK on linux. If this is undefined, it's probably because
-	 * you forgot to define _GNU_SOURCE at the top of this file.
+#if defined(POSIX_SPAWN_USEVFORK) || defined(__GLIBC__)
+	/* Force USEVFORK on GNU libc. If this is undefined, it's probably
+	 * because you forgot to define _GNU_SOURCE at the top of this file.
 	 */
 	flags |= POSIX_SPAWN_USEVFORK;
 #endif

